src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -418,9 +418,8 @@ public UserJourneyResult call() {
       long startTime = System.nanoTime();
       try {
         testScript.userJourney(testUser);
-      } catch (Exception ex) {
-        result.exceptionThrown = ex;
-        ex.printStackTrace(); // print the stack trace to the console
+      } catch (Throwable ex) {
+        result.saveExceptionThrown(ex);
```

Comment:
Why don't we want the stack trace here?

Reply:
We do. I just moved it into the `saveExceptionThrown` method because I think we always want to log the stack trace.